README.md

```diff
@@ -201,6 +202,7 @@ Docker EE is on the same code base as Docker CE, so also built from Moby, with c
 * [Dockerfile Project](http://dockerfile.github.io/) : Trusted Automated Docker Builds. Dockerfile Project maintains a central repository of Dockerfile for various popular open source software services runnable on a Docker container.
 * [Dockerfile Example](https://github.com/komljen/dockerfile-examples) by [@komljen](https://github.com/komljen)
 * [Dockerfile Example 2](https://github.com/kstaken/dockerfile-examples) by [@kstaken](https://github.com/kstaken)
+* [Docker Multi-stage build guide](https://codefresh.io/docker-guides/docker-multi-stage-builds/) by [Codefresh](https://github.com/codefresh-io)
```

Comment:
Under this category, we're looking for Dockerfile examples, not how to write one. In addition, there are plenty of tutorials out there.